GAMEFLOW.md

```diff
@@ -0,0 +1,1306 @@
+  </tr>
+  <tr>
+    <td>90</td>
+    <td>Magnum ammo</td>
+  </tr>
+  <tr>
+    <td>91</td>
+    <td>Uzi ammo</td>
+  </tr>
+  <tr>
+    <td>93</td>
+    <td>Small medipack</td>
+  </tr>
+  <tr>
+    <td>94</td>
+    <td>Large medipack</td>
+  </tr>
+  <tr>
+    <td>110</td>
+    <td>Puzzle1</td>
+  </tr>
+  <tr>
+    <td>111</td>
+    <td>Puzzle2</td>
+  </tr>
+  <tr>
+    <td>112</td>
+    <td>Puzzle3</td>
+  </tr>
+  <tr>
+    <td>113</td>
+    <td>Puzzle4</td>
+  </tr>
+  <tr>
+    <td>126</td>
+    <td>Lead bar</td>
+  </tr>
+  <tr>
+    <td>129</td>
+    <td>Key1</td>
+  </tr>
+  <tr>
+    <td>130</td>
+    <td>Key2</td>
+  </tr>
+  <tr>
+    <td>131</td>
+    <td>Key3</td>
+  </tr>
+  <tr>
+    <td>132</td>
+    <td>Key4</td>
+  </tr>
+  <tr>
+    <td>141</td>
+    <td>Pickup1</td>
+  </tr>
+  <tr>
+    <td>142</td>
+    <td>Pickup2</td>
+  </tr>
+  <tr>
+    <td>144</td>
+    <td>Scion (à la Pierre)</td>
+  </tr>
+</table>
+
+## Injections
+Injections defined in the global gameflow will by default be applied to each
+level. Individual levels can also specify their own specific injections to
+include. In that case, the global injections are merged with the level's when
+the level loads.
+
+Individual levels can set `inherit_injections` to `false` and as a result they
+will not use the global injection files. If those levels have their own
+injections defined, only those will be used. And of course, if they have none
+defined, nothing will be injected.
+
+_Disclaimer: Custom levels should not use the injections mechanism and instead
+should provide data that is already correct and consistent. Reports of bugs
+about injection files not working for custom levels will not be considered. Note
+however that the injections that relate to Lara can work in custom levels
+that do not modify Lara's default mesh structure or animations. These injection
+files are based on the original Lara model._
+
+The gameflow will ignore referenced injection files that do not exist, but it is
+best practice to remove the references to maintain a clean gameflow file.
+
+Following is a summary of what each of the default injection files that are
+provided with the game achieves.
+
+<table>
+  <tr valign="top" align="left">
+    <th>Injection file</th>
+    <th>Purpose</th>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>*_fd.bin</code>
+    </td>
+    <td>
+      Injects fixes for floor data issues in the original levels. Refer to the
+      README for a full list of fixes.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>*_itemrots.bin</code>
+    </td>
+    <td>
+      Injects rotations on pickup items so they make more visual sense when
+      using the 3D pickups option.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>*_textures.bin</code>
+    </td>
+    <td>
+      Injects various texture fixes, such as gaps in the walls or wrongly
+      colored models. Refer to the README for a full list of fixes.
```

Comment:
Maybe add `in the original levels` here too?

Reply:
Thanks, keeps it consistent 👍